package.json

```diff
@@ -9,6 +9,7 @@
     "deploy-studio": "graph deploy --studio snapshot",
     "deploy-studio-sepolia": "graph deploy --studio snapshot-sepolia subgraph.sepolia.yaml",
     "deploy-studio-base": "graph deploy --studio snapshot-base subgraph.base.yaml",
+    "deploy-studio-blast-mainnet": "graph deploy --studio snapshot-blast-mainnet subgraph.blast-mainnet.yaml",
```

Comment:
Can you call it simply snapshot-blast ?

Reply:
There is a testnet too, using the same name as mentioned here https://thegraph.com/docs/en/developing/supported-networks/

Reply:
Ok weird, others have testnet aswell but dont add `-mainnet` in the name. But ok let's use what's subgraph uses